pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -922,7 +918,7 @@ func (t *Test) choosePreviousReleases() ([]*clusterupgrade.Version, error) {
 	// function to change in case the rules around what upgrades are
 	// possible in CRDB change.
 	possiblePredecessorsFor := func(v *clusterupgrade.Version) ([]*clusterupgrade.Version, error) {
-		pred, err := t.predecessorFunc(t.prng, v, t.options.minimumSupportedVersion)
```

Comment:
Yikes. I bet this wouldn't have happened if the option had been lifted into a var outside of the closure; e.g., `userPredecessorFunc := t.options.predecessorFunc`.